app/views/components/_datetime_fields.html.erb

```diff
@@ -1,6 +1,8 @@
 <%
  prefix = prefix
  field_name = field_name
+ date_only ||= date_only || false
+ date_heading = date_heading || nil
```

Comment:
Isn't this the same as:

```suggestion
 date_only ||= false
 date_heading ||= nil
```

Reply:
Hmm, I'm not too sure. I understood this to be a means of gaining efficiency though on reflection I'm not sure that stacks up. I guess those should both be consistent at least.

Reply:
This is updated as in the comment above for consistency.